iroh-rpc/Cargo.toml

```diff
@@ -6,3 +6,16 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+rkyv = { version = "0.7.37", features = ["std", "alloc", "validation"] }
+libp2p = "0.43.0"
+tokio = { version = "1.17.0", features = ["full"] }
+futures = "0.3.21"
+async-trait = "0.1.53"
+bytecheck = "0.6.7"
+smallvec = "1.8.0"
+rand = "0.8.5"
+log = "0.4.16"
+pretty_env_logger = "0.4.0"
```

Comment:
we are using `tracing` and `tracing_subscriber` for logging & tracing now, see `iroh-p2p` for an example that behaves similarly